core/controllers/suggestion_test.py

```diff
@@ -980,9 +985,14 @@ def test_update_suggestion_updates_question_suggestion_content(self):
             feconf.UPDATE_QUESTION_SUGGESTION_URL_PREFIX,
             suggestion.suggestion_id), {
                 'question_state_data': question_state_data,
-                'skill_difficulty': 0.6
-            }, csrf_token=csrf_token, upload_files=(
-                ('img.png', 'img.png', raw_image),))
+                'skill_difficulty': 0.6,
+                'files': {
```

Comment:
SInce images for Question Suggestions and Question Suggestion Update are sent prior to submit the questions(untill this is implemented https://github.com/oppia/oppia/issues/14298), this is probably not needed.

Reply:
Done.
https://user-images.githubusercontent.com/37933427/143822344-5c4b88e8-0ef1-41ab-b608-ccb4f4c0ecad.mp4

Reply:
PTAL @chris7716